data.js

```diff
@@ -0,0 +1,13 @@
+var axios = require('axios');
```

Comment:
@cofuente Oversight. It can go in `client` since it's technically client-specific. The Node-related JS files I was keeping in the home directly for ease of access while creating them and never moved them. I'll do the same for `post.js` which hasn't been committed yet.

Reply:
Awesome, thank you!